ui/src/consts/regex.ts

```diff
@@ -0,0 +1,6 @@
+export const regex = {
+  email: new RegExp(
+    /^(([^<>()[\]\\.,;:\s@"]+(\.[^<>()[\]\\.,;:\s@"]+)*)|(".+"))@((\[[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}\.[0-9]{1,3}])|(([a-zA-Z\-0-9]+\.)+[a-zA-Z]{2,}))$/
+  ),
+  password: new RegExp(/^(?=.*[A-Z])(?=.*[a-z])(?=.*[0-9])(?=.*[!@#$%^&*])[A-Za-z0-9!@#$%^&*]{8,128}$/),
```

Comment:
to moglibyśmy stworzyć przy pomocy yupa.

yup.string().min(8, 'hasło musi mieć przynajmniej 8 znaków').max(200).matches(/tutaj regex tylko na sprawdzenie czy ma znak specjalny)